pandas/io/tests/test_gbq.py

```diff
@@ -56,6 +56,27 @@ def _skip_if_no_private_key_contents():
         _skip_if_no_private_key_contents()
 
 
+def _skip_if_get_correct_default_credentials(can=False):
+    got_credentials = False
+    try:
+        from oauth2client.client import GoogleCredentials
+        from apiclient.discovery import build
+        credentials = GoogleCredentials.get_application_default()
+        bigquery_service = build('bigquery', 'v2', credentials=credentials)
+        jobs = bigquery_service.jobs()
+        job_data = {'configuration': {'query': {'query': 'SELECT 1'}}}
+        jobs.insert(projectId=PROJECT_ID, body=job_data).execute()
+        got_credentials = True
+    except:
+        pass
```

Comment:
why do you need the ``got_credentials``, if it hits the ``except ImportError`` you can just skip at that point.